src/components/elements/auto-carousel.js

```diff
@@ -1,92 +1,115 @@
 import React from 'react'
-import styled from 'styled-components'
+import styled, { keyframes } from 'styled-components'
 import PropTypes from 'prop-types'
-import Container from 'components/containers/container'
 
-const CarouselContainer = styled(Container)`
+const AutoCarouselSection = styled.section`
+    width: ${props => props.width};
+    display: flex;
+    justify-content: flex-start;
     overflow: hidden;
-    justify-content: space-between;
-
-    div {
-        transition: transform ${props => (props.transition ? '1s' : '0')};
-        transform: translate(${props => props.position}rem);
+    margin: 0 auto;
+`
+const move_items = count_child => keyframes`
+    0% {
+        transform: translateX(0);
+    }
+    100% {
+        transform: translateX(-${count_child}%);
     }
 `
-const CarouselWrapper = styled.div`
-    max-width: 9.6rem;
-    margin-right: 3.2rem;
+const ItemContainer = styled.div`
+    padding: 0 ${props => props.padding / 2}px;
+    animation-name: ${props =>
+        props.should_carousel_move ? move_items(props.count_child * 100) : ''};
+    animation-duration: ${props => props.transition_duration}ms;
+    animation-fill-mode: both;
+    animation-iteration-count: infinite;
+    animation-timing-function: linear;
```

Comment:
Can you try moving this to the parent `AutoCarouselSection`? should fix the "shakiness"

Reply:
Let me try your suggestion, I'll tell you the result.